developer/src/kmc-package/src/kmp-compiler.ts

```diff
@@ -18,7 +19,8 @@ export default class KmpCompiler {
   constructor(private callbacks: CompilerCallbacks) {
   }
 
-  public transformKpsToKmpObject(kpsString: string, kpsPath: string): KmpJsonFile {
+  // TODO: load the file here via a callback, rather than passing it in as a string
```

Comment:
// TODO: load the file here via a callback, rather than passing it in as a string